.github/workflows/ci-cd.yml

```diff
@@ -50,20 +50,22 @@ jobs:
       - name: Update the changelog
         run: inv update-changelog --sha=$GITHUB_SHA
 
-      - name: Get release metadata
-        id: metadata
-        run: |
-         echo "release_tag=`inv get-release-tag`" >> "$GITHUB_OUTPUT"
-         echo "git_hash=`git rev-parse HEAD`" >> "$GITHUB_OUTPUT"
+      - name: Get release tag
+        id: tag
+        run: echo "release_tag=`inv get-release-tag`" >> "$GITHUB_OUTPUT"
 
       - uses: EndBug/add-and-commit@a94899bca583c204427a224a7af87c02f9b325d5 # v9
         with:
           add: modal_version/__init__.py CHANGELOG.md
-          tag: ${{ steps.metadata.outputs.release_tag }}
+          tag: ${{ steps.tag.outputs.release_tag }}
```

Comment:
There's still a problem here which is that this doesn't push the tag when there's nothing to commit. Not sure if that's a configuration I can set or if we're going to need to drop down to manual git (which seems fine in principle but I expect it will involve more thrashing with CI to get it right 😢).